Fearless Concurrency/Threads/Using Message Passing to Transfer Data Between Threads/task.md

```diff
@@ -108,16 +108,16 @@ The ownership rules play a vital role in message sending because they help you w
 Here, we try to print `val` after we’ve sent it down the channel via `tx.send`. Allowing this would be a bad idea: once the value has been sent to another thread, that thread could modify or drop it before we try to use the value again. Potentially, the other thread’s modifications could cause errors or unexpected results due to inconsistent or nonexistent data. However, Rust gives us an error if we try to compile the code in the example below:
 
 ```text
-    error[E0382]: use of moved value: `val`
+    error[E0382]: borrow of moved value: `val`
       --> src/main.rs:10:31
        |
+    8  |         let val = String::from("hi");
+       |             --- move occurs because `val` has type `String`, which does not implement the `Copy` trait
```

Comment:
--- move occurs because `val` is of type `String`, which does not implement the `Copy` trait

Reply:
This is a part of the compiler message. We should definitely find it and make a pull request into the rust source.